TypeCobol/Compiler/Diagnostics/Resources/DiagnosticMessages.csv

```diff
@@ -13,7 +13,7 @@ Category;Code;Severity;MessageTemplate;Document;PageNumber;ReferenceText
 2;12;2;A blank was missing before character "{0}" in column {1}. A blank was assumed.;0;0;
 2;13;1;Invalid character found after + : expecting space, digit or .;1;0;Syntax described pages : 37, 38, 254
 2;14;1;Invalid character found after - : expecting space, digit or .;1;0;Syntax described pages : 37, 38, 254
-2;15;1;Invalid character found after pseudo-text delimiter == : expecting space, comma, semicolon or period;1;47;A closing pseudo-text delimiter must be immediately followed by a separator space, comma, semicolon, or period
+2;15;1;Invalid character found after pseudo-text delimiter == : "{0}";1;47;A closing pseudo-text delimiter should be followed by a space separator.
```

Comment:
It'd be easier for our Cobol developers to read `==:` instead of `== :`.

Reply:
done